### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by d-damien**

### CODE REVIEW

**Comment by d-damien** [comment]
`<em>` tags not working ?

**Comment by kettanaito** [comment]
Hey, @d-damien. Thank you so much for preparing this! Let me test it out and let you know if I find anything.

**Comment by kettanaito** [comment]
Seems like it doesn't like the single quote before the `<em>` tag. Let's try removing it locally... 

It handles the `<em`> tags correctly if:

- There's a space between `d'` and `<`: `d' <em>hash</em>`
- There's an escape character before `<`: `d'\\<em>` (however one escape symbol is considered invalid, and two result in a`\` backward slash in the text).

@d-damien, is the space option a viable solution in regard to French grammar?

File: src/locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "url.http.path.name": "Chemin",
+  "url.http.path.definition": "Le chemin est une chaîne de caractères désignant une ressource sur un serveur.",
+  "url.http.path.description": "<p>Le serveur, ou le client, décide de comment gérer une requête sur un chemin donné. Par exemple, il peut retourner une page HTML, ou communiquer avec un autre serveur et répondre avec des données brutes.</p>",
+  "url.http.query.name": "Paramètre de requête",
+  "url.http.query.definition": "Les paramètres de requête sont des arguments du chemin.",
+  "url.http.query.description": "<p>Ils sont stockés dans des paires <code>clef=valeur</code> qui sont ajoutées au chemin après un <code>?</code> (point d'interrogation). Chaque paramètre suivant est séparé par un <code>&</code> (esperluette) : </p><ul><li><code>?utilisateur=123&type=lecteur</code></li></ul><p>Le serveur peut analyser une URL afin d'obtenir des informations supplémentaires sur la requête.</p>",
```
Comment (kettanaito) [inline src/locales/fr.json]: May I ask you to preserve English query parameters? They are not localized in any of the present languages, so it'd be nice to keep the translations consistent. Thanks!

Comment (d-damien) [reply src/locales/fr.json]: done

**Comment by d-damien** [comment]
I had to use NBSP so as to avoid an ugly line break.

Btw howurls.work is a cool tool to teach.

**Review by kettanaito — APPROVED** [review]
This is awesome! Thank you so much for your contribution, @d-damien 🎉

**Comment by kettanaito** [comment]
Thank you for making this website much more awesome!

**Pull request merged**